### DISCUSSION TITLE
Touch screen?

### DISCUSSION THREAD

**Discussion by MatyxCZ** [mriscoc/Ender3V2S1#1317] [Q&A] [answered]
I just want to ask can it work with touch screens such as this one https://www.3djake.cz/3d-tiskarny-nahradni-dily-creality/touchscreen-upgrade?sai=10689&gad_source=1&gclid=CjwKCAjwmrqzBhAoEiwAXVpgonxaBMCJBmf4qE5XE4BClx8muYEsIaTzbNdmaJ6N1_ueXJKvvd4XGhoCUw8QAvD_BwE ? sorry its a Czech website

**Comment by mriscoc** [comment, marked as answer]
No, this firmware is not compatible with that display unit.

**Reply by MatyxCZ** [reply]
Thanks i was thinking about buying it but if there is no profesional firmware support i aint buying hehe.
